src/containers/Chat/ChatMessages/ChatMessage/LocationRequestTemplate/LocationRequestTemplate.tsx

```diff
@@ -29,6 +20,7 @@ export const LocationRequestTemplate = ({
   onSendLocationClick = () => {},
 }: LocationRequestTemplateProps) => {
   const body = content.body.text;
+  const locationPayload = SAMPLE_MEDIA_FOR_SIMULATOR[5];
```

Comment:
We just need to send the payload part of SAMPLE_MEDIA. I tried this in a flow and its not working as expected.